### DISCUSSION TITLE
AppImage run troubles

### DISCUSSION THREAD

**Discussion by thefrankk** [AppImage/AppImageKit#1309] [Q&A]
Hi, I have a problem, I built my AppImage and it works perfect, the problem is when I deleted the root folder (where I made the build) now, I can't open it anymore, throws errors like "..../release/bundle: No such file or directory (This is all the folders that i deleted)

I was sure when I build the AppImage, I will be able to run it in every system.

This is the correct behavior or I missing something?
Thank you.

**Comment by probonopd** [comment]
Maybe your application (or somethin inside the AppImage) tries to load something from an absolute path that was hardcoded at build time. Need to make the software relocatable in the filesystem.

If you can share the exact error message and the source code in question, we might be able to help you better.

**Reply by thefrankk** [reply]
Hi! Thank you for your reply. I'm trying to use it with a Flutter build. Based on that you said I will check for that absolute path and try to change it.
